lib/LambdaHelper.js

```diff
@@ -78,7 +80,51 @@ var _lambdaizeModule = function(handlerName, externals) {
     return outputModuleCode;
 }
 
-var _createProxy = function (executionStore, queueInitializedPromise, uploadPromise) {
+var _createFeedbackRoot = function(feedbackStore, functionName, requestId, callback) {
+    var entry       = { requestId: requestId, callback: callback, startedAt: new Date().getTime() },
+        requestKey  = 'req-' + requestId,
+        hoseKey     = functionName + '.hose',
+        timerKey    = functionName + '.timer';
+
+    if(!feedbackStore.hasOwnProperty(functionName)) {
+        
+        var src = new hose.Source({
+            aws: _awsConfig,
+            LogGroup: '/aws/lambda/' + functionName,
+        });
+
+        src.on('logs', function(batch) {
+            for(var i in batch) {
+                var log = batch[i];
+                // Timeouts handling
+                if(log.message.match(/Task timed out after/gi)) {
+                    feedbackStore[requestKey].call(null, new Error(log.message.trim()));    
+                }
+            }
+        });
+
+        src.on('error', function(error) {
+            // TODO Handle errors gracefully
+            console.log('Clouwatch hose error -->', error);
+        });
+
+        feedbackStore[functionName] = [];
+        feedbackStore[hoseKey] = src;
+    }
+
+    feedbackStore[requestKey] = callback;
+    feedbackStore[functionName].push(entry);
```

Comment:
makes no sens to me why feedbackStore[functionName] is an array ... except if your hose poll feedback many times and you want to store the feedback ... 

Is it necessary to keep the feedback in memory ?